Cargo.toml

```diff
@@ -48,6 +50,5 @@ optional = true
 version = "0.1.4"
 optional = true
 
-[build-dependencies]
-phf_generator = "0.7.4"
-phf_shared = "0.7.4"
+[build-dependencies.string_cache_codegen]
+version = "0.2.11"
```

Comment:
It turns out I didn't understand what `path` dependencies do (https://github.com/rust-lang/cargo/pull/2625 - when pulling from crates.io, `path =` are ignored).

I've added `path` (so tests will work) and left in the `exclude` (there is no point in users getting the `string-cache-codegen` folder because Cargo will ignore it).